--- MDB2_TableBrowser-0.1.0/usr/share/pear/MDB2/TableBrowser/Single.php~	2008-12-27 02:19:34.659650805 +0100
+++ MDB2_TableBrowser-0.1.0/usr/share/pear/MDB2/TableBrowser/Single.php	2008-12-27 02:34:32.732269122 +0100
@@ -12,13 +12,13 @@
  * @version   SVN:<svn_id>
  * @link      http://code.google.com/p/mdb2tablebrowser/source/browse/trunk/MDB2_TableBrowser/TableBrowser/DBException.php
  */
-require_once 'Interfaces.php';
-require_once 'ParameterException.php';
-require_once 'DBException.php';
-require_once 'TableValidator.php';
-require_once 'FilterManager.php';
-require_once 'ColumnManager.php';
-require_once 'MultipleFliterManager.php';
+require_once 'MDB2/TableBrowser/Interfaces.php';
+require_once 'MDB2/TableBrowser/ParameterException.php';
+require_once 'MDB2/TableBrowser/DBException.php';
+require_once 'MDB2/TableBrowser/TableValidator.php';
+require_once 'MDB2/TableBrowser/FilterManager.php';
+require_once 'MDB2/TableBrowser/ColumnManager.php';
+require_once 'MDB2/TableBrowser/MultipleFliterManager.php';
 
 
 /**
--- MDB2_TableBrowser-0.1.0/usr/share/pear/MDB2/TableBrowser/FilterManager.php~	2008-12-27 02:35:25.003723218 +0100
+++ MDB2_TableBrowser-0.1.0/usr/share/pear/MDB2/TableBrowser/FilterManager.php	2008-12-27 02:37:05.466212139 +0100
@@ -13,9 +13,9 @@
  * @link      http://code.google.com/p/mdb2tablebrowser/source/browse/trunk/MDB2_TableBrowser/TableBrowser/FilterManager.php
  */
 
-require_once 'Interfaces.php';
+require_once 'MDB2/TableBrowser/Interfaces.php';
 require_once 'MDB2.php';
-require_once 'DBException.php';
+require_once 'MDB2/TableBrowser/DBException.php';
 
 /**
  * This class implements the whereClauseGenerator interface
@@ -165,4 +165,4 @@
         }
     }
 }
-?>
\ Brak znaku nowej linii na końcu pliku
+?>
--- MDB2_TableBrowser-0.1.0/usr/share/pear/MDB2/TableBrowser/MultipleFliterManager.php~	2008-12-27 02:35:25.003723218 +0100
+++ MDB2_TableBrowser-0.1.0/usr/share/pear/MDB2/TableBrowser/MultipleFliterManager.php	2008-12-27 02:37:38.872875418 +0100
@@ -13,10 +13,10 @@
  * @link      http://code.google.com/p/mdb2tablebrowser/source/browse/trunk/MDB2_TableBrowser/TableBrowser/FilterManager.php
  */
 
-require_once 'Interfaces.php';
+require_once 'MDB2/TableBrowser/Interfaces.php';
 require_once 'MDB2.php';
-require_once 'DBException.php';
-require_once 'FilterManager.php';
+require_once 'MDB2/TableBrowser/DBException.php';
+require_once 'MDB2/TableBrowser/FilterManager.php';
 
 /**
  * This class implements the MDB2_TableBrowser_InterfaceMultipleWhereClauseGenerator interface
@@ -193,4 +193,4 @@
         $whereClause = join(' OR ', $sqlClauses);
         return $whereClause;
     }
-}
\ Brak znaku nowej linii na końcu pliku
+}
